### PR TITLE
Refine the error messages in devicetwin/dttype

### PR DESCRIPTION
**Pull request by sailorvii**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
NONE

### CODE REVIEW

**Comment by fisherxu** [comment]
lgtm, would you like to squash the commits?

**Comment by daixiang0** [comment]
Better to use a more meaningful title.

**Comment by sailorvii** [comment]
> Better to use a more meaningful title.

Yes, I just new a branch named oneword and push the change. Not clear why it's used by the pull request message.

**Comment by kadisi** [comment]
@sailorvii please squash the commits

File: edge/pkg/devicetwin/dttype/types.go [hunk 1]
```diff
@@ -29,6 +29,13 @@ type BaseMessage struct {
 	Timestamp int64  `json:"timestamp"`
 }
 
+const (
+	ErrorUnmarshalInfo = "Unmarshal update request body failed, please check the request"
+	ErrorUpdateInfo    = "Update twin error, key:twin does not exist"
+	ErrorKeyInfo       = "The key of twin must only include upper or lowercase letters, number, english, and special letter - _ . , : / @ # and the length of key should be less than 128 bytes"
+	ErrorValueInfo     = "The value of twin must only include upper or lowercase letters, number, english, and special letter - _ . , : / @ # and the length of value should be less than 512 bytes"
```
Comment (kadisi) [inline edge/pkg/devicetwin/dttype/types.go:36]: Suggest using `errors. Errorf ()`

Comment (sailorvii) [reply edge/pkg/devicetwin/dttype/types.go]: Thanks for your comments.
Done.

**Pull request merged**